src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java

```diff
@@ -257,17 +250,6 @@ public void onTraversalStart() {
             throw new UserException("VCF has no genotypes");
         }
 
-        if ( header.hasInfoLine(GATKVCFConstants.MLE_ALLELE_COUNT_KEY) ) {
-            final VCFInfoHeaderLine mleLine = header.getInfoHeaderLine(GATKVCFConstants.MLE_ALLELE_COUNT_KEY);
-            if ( mleLine.getCountType() != VCFHeaderLineCount.A ) {
-                throw new UserException("VCF does not have a properly formatted MLEAC field: the count type should be \"A\"");
```

Comment:
There's no regression test for this, but it's from a particular issue: https://github.com/broadinstitute/gsa-unstable/issues/845
The issue was caused by a 4.0 VCF as the supplemental VCF.

Reply:
Hmn, I think I thought this was being checked somewhere else now, but I can't find any evidence that it is.  Not sure why I took that out...